Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -238,25 +239,23 @@ namespace O3DE::ProjectManager
 
     void PythonBindings::OnStdOut(const char* msg)
     {
-        AZStd::string message{ msg };
-        // escape % characters by using %% in the format string to avoid crashing
-        AZ::StringFunc::Replace(message, "%", "%%", true /* case sensitive since it is faster */);
-        AZ_TracePrintf("Python", message.c_str());
+        AZ::Debug::Trace::Output("Python", msg);
     }
 
     void PythonBindings::OnStdError(const char* msg)
     {
-        AZStd::string lastPythonError = msg;
+        AZStd::string_view lastPythonError{ msg };
         constexpr const char* pythonErrorPrefix = "ERROR:root:";
         constexpr size_t lengthOfErrorPrefix = AZStd::char_traits<char>::length(pythonErrorPrefix);
         auto errorPrefix = lastPythonError.find(pythonErrorPrefix);
         if (errorPrefix != AZStd::string::npos)
         {
-            lastPythonError.erase(errorPrefix, lengthOfErrorPrefix);
+            lastPythonError = lastPythonError.substr(lengthOfErrorPrefix, lastPythonError.length() - lengthOfErrorPrefix);
```

Comment:
The second argument to substr isn't needed.
By default substr goes to the end of the string
This can be simplified to
```suggestion
        AZStd::string_view lastPythonError{ msg };
        if (constexpr AZStd::string_view pythonErrorPrefix = "ERROR:root:";
            lastPythonError.starts_with(pythonErrorPrefix))
        {
            lastPythonError.substr(pythonErrorPrefix.size());
        }
```